CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ HunterGate(
     SHA1 "29f10683f10c7b35e1f599d71542af0c2daa6a01"
 )
 
-project(jaegertracing VERSION 0.5.0)
+project(jaegertracing VERSION 0.5.1dev)
```

Comment:
@mdouaihy this didn't work for me, hunter complained
```
CMake Error at CMakeLists.txt:22 (project):
  VERSION "0.5.1dev" format invalid.
```
Is there another way to specify that the version is not "official"?

Reply:
@yurishkuro, this is CMake complaining about the version of the project and not Huinter.

According to the [documentation](https://cmake.org/cmake/help/v3.15/variable/CMAKE_VERSION.html), "0.5.1dev" is not a valid version.

Reply:
According to docs:

> Development versions of CMake use the format: `<major>.<minor>.<date>[-<id>]`
> where the <date> component is of format CCYYMMDD and <id> may contain arbitrary text. 

it looks like `0.5.20190908-dev` should be valid, but it still fails